bin/Powheg/production/pre2017/13TeV/dijet_CT14NLO_13TeV/dijet_CT14NLO_13TeV_powheg.input

```diff
@@ -0,0 +1,91 @@
+numevts NEVENTS        ! number of events to be generated
+ih1  1               ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2  1               ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500d0        ! energy of beam 1
+ebeam2 6500d0        ! energy of beam 2
+
+bornktmin 5d0       ! (default 0d0) Generation cut: minimum kt in underlying Born
+bornsuppfact 300d0  ! (default 0d0) Mass parameter for Born suppression factor.
+                     ! If < 0 suppfact = 1.
+
+
+! To be set only if using internal (mlm) pdfs
+! 131 cteq6m
+ndns1 131            ! pdf set for hadron 1 (mlm numbering)
+ndns2 131            ! pdf set for hadron 2 (mlm numbering)
+
+! To be set only if using LHA pdfs
+! 13100 CT14NLO
+lhans1  13100      ! pdf set for hadron 1 (LHA numbering)
+lhans2  13100      ! pdf set for hadron 2 (LHA numbering)	
+
+! To be set only if using different pdf sets for the two incoming hadrons
+#QCDLambda5  0.25    ! for not equal pdf sets 
+
+renscfact  1d0      ! (default 1d0) ren scale factor: muren  = muref * renscfact 
+facscfact  1d0      ! (default 1d0) fac scale factor: mufact = muref * facscfact 
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1    ! If 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1    ! If 1 use norm of upper bounding function stored
+                     ! in pwgubound.dat, if present; <> 1 regenerate
+
+! A typical call uses 1/1400 seconds (1400 calls per second)
+ncall1 20000         ! No. calls for the construction of the importance sampling grid
+itmx1 5              ! No. iterations for grid: total 100000 calls ~ 70 seconds
+ncall2 20000         ! No. calls for the computation of the upper bounding
+                     ! envelope for the generation of radiation
+itmx2 5              ! No. iterations for the above
+
+! Notice: the total number of calls is ncall2*itmx2*foldcsi*foldy*foldphi
+! these folding numbers yield a negative fraction of 0.5% with bornktmin=10 GeV.
+! With these settings: ncall2*itmx2*foldcsi*foldy*foldphi=5M, 60 minutes
+foldcsi 5            ! No. folds on csi integration
+foldy   10            ! No. folds on  y  integration
```

Comment:
do you really need such an high folding? did you copy it from somewhere else?

Reply:
Hi Luca, This datacard was provided to me by analyzer. I asked him regarding this and got following answer :

For Powheg for 7 TeV was generated based on the configuration on the Atlas paper [1].
For 13 TeV I imagine that will be good to use the same configuration.
The negative cross sections can be avoided using the option [2]:

withnegweights 0     ! (default 0). If 1 use negative weigh

My experienceto produce private sample at 7 TeV show that the steps before simulate the "collision event" (where interaction energy is "selected" and the daughters particles are created) that around 1.5~2 hour, the remaining time is producing and saving events on the .lhe file.

[1]
page 4
https://journals.aps.org/prd/pdf/10.1103/PhysRevD.86.014022

[2]
numevts 1000000        ! number of events to be generated
ih1  1               ! hadron 1 (1 for protons, -1 for antiprotons)
ih2  1               ! hadron 2 (1 for protons, -1 for antiprotons)
ebeam1 3500d0        ! energy of beam 1
ebeam2 3500d0        ! energy of beam 2

bornktmin 5d0       ! (default 0d0) Generation cut: minimum kt in underlying Born
#
bornsuppfact 300d0  ! (default 0d0) Mass parameter for Born suppression factor.
                     ! If < 0 suppfact = 1.


! To be set only if using internal (mlm) pdfs
! 131 cteq6m
!ndns1 131            ! pdf set for hadron 1 (mlm numbering)
!ndns2 131            ! pdf set for hadron 2 (mlm numbering)

! To be set only if using LHA pdfs
! 10050 cteq6m
! 10550 cteq66
! 10800 CT10
! 11000 CT10NLO
! 11200 CT10NNLO
! 13200 CT14LO
! 13100 CT14NLO
! 13000 CT14NNLO

 lhans1  13100      ! pdf set for hadron 1 (LHA numbering)
 lhans2  13100      ! pdf set for hadron 2 (LHA numbering)   

! To be set only if using different pdf sets for the two incoming hadrons
#QCDLambda5  0.25    ! for not equal pdf sets

#renscfact  1d0      ! (default 1d0) ren scale factor: muren  = muref * renscfact
#facscfact  1d0      ! (default 1d0) fac scale factor: mufact = muref * facscfact

! Parameters to allow or not the use of stored data
use-old-grid    1    ! If 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
use-old-ubound  1    ! If 1 use norm of upper bounding function stored
                     ! in pwgubound.dat, if present; <> 1 regenerate

! A typical call uses 1/1400 seconds (1400 calls per second)
ncall1 20000         ! No. calls for the construction of the importance sampling grid
itmx1 5              ! No. iterations for grid: total 100000 calls ~ 70 seconds
ncall2 20000         ! No. calls for the computation of the upper bounding
                     ! envelope for the generation of radiation
itmx2 5              ! No. iterations for the above

! Notice: the total number of calls is ncall2*itmx2*foldcsi*foldy*foldphi
! these folding numbers yield a negative fraction of 0.5% with bornktmin=10 GeV.
! With these settings: ncall2*itmx2*foldcsi*foldy*foldphi=5M, 60 minutes
foldcsi 5            ! No. folds on csi integration
foldy   10            ! No. folds on  y  integration
foldphi 2            ! No. folds on phi integration

nubound 5000000       ! No. calls to set up the upper bounding norms for radiation.
                     ! This is performed using only the Born cross section (fast)

! OPTIONAL PARAMETERS

withnegweights 0     ! (default 0). If 1 use negative weights.
#bornonly 1          ! (default 0). If 1 compute underlying Born using LO
                     ! cross section only.

#ptsqmin    0.8      ! (default 0.8 GeV) minimum pt for generation of radiation
#charmthr   1.5      ! (default 1.5 GeV) charm treshold for gluon splitting
#bottomthr  5.0      ! (default 5.0 GeV) bottom treshold for gluon splitting
#testplots  1        ! (default 0, do not) do NLO and PWHG distributions
#charmthrpdf  1.5    ! (default 1.5 GeV) pdf charm treshold
#bottomthrpdf 5.0    ! (default 5.0 GeV) pdf bottom treshold

#xupbound 2d0        ! increase upper bound for radiation generation

#iseed    5421       ! Start the random number generator with seed iseed
#rand1     0         ! skipping  rand2*100000000+rand1 numbers (see RM48
#rand2     0         ! short writeup in CERNLIB).             
#manyseeds 1         ! Used to perform multiple runs with different random
                     ! seeds in the same directory.
                     ! If set to 1, the program asks for an integer j;
                     ! The file pwgseeds.dat at line j is read, and the
                     ! integer at line j is used to initialize the random
                     ! sequence for the generation of the event.
                     ! The event file is called pwgevents-'j'.lhe

doublefsr 1          ! Default 0; if 1 use new mechanism to generate regions
                     ! such that the emitted harder than the
                     ! emitter in FSR is suppressed. If doublefsr=0 this is
                     ! only the case for emitted gluons (old behaviour). If
                     ! 1 it is also applied to emitted quarks.
                     ! If set, it strongly reduces spikes on showered output.



#par_diexp 4         ! default is 2. With 4 there is a stronger separation
#par_dijexp 4        ! of regions, it may help to reduce spikes when generating
#par_2gsupp 4        ! weighted events.


#manyseeds 1
#multigrid 0

Reply:
please use
withnegweights 1
because the removal of events with negative weigths is not recommended...